clusterloader2/pkg/prometheus/experimental.go

```diff
@@ -19,17 +19,23 @@ package prometheus
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/spf13/pflag"
-	"k8s.io/apimachinery/pkg/util/wait"
 	"os/exec"
 	"time"
 
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/wait"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog"
 )
 
+const (
+	kubeProxyManifest = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/default/experimental/prometheus-serviceMonitorKubeProxy.yaml"
+)
+
 var (
 	shouldSnapshotPrometheusDisk = pflag.Bool("experimental-gcp-snapshot-prometheus-disk", false, "(experimental, provider=gce|gke only) whether to snapshot Prometheus disk before Prometheus stack is torn down")
+	enableKubeProxyScrapping     = pflag.Bool("enable-kube-proxy-scrapping", false, "whether to scrape kube-proxy")
```

Comment:
Two things:

1. I don't like, having it in the "experimental" file. It was already enabled in 5K and worked, it's most likely gonna work again. Having it in experimental doesn't bring us any gain, and on the other hand we will have to move it away from here once we confirm it works. Let's move it to the prometheus.go file

2. I don't like having it flag gated. In my opinion, whether we monitor kube proxy or not is specific to the test scenario itself, not the the mechanics of clusterloader2. Similar to whether we monitor etcd or whether we run some measurement or not.  I believe it would be better if we introduced a variable, e.g. PROMETHEUS_SCRAPE_KUBEPROXY, that user can set in his test scenario (similar to already existing PROMETHEUS_SCRAPE_ETCD). I'm aware of the problem of failures when applying empty manifests, but I think it could be solved some other way. E.g. by checking if manifests is empty before sending request via k8s-client. It would be a more generic solution, as it would allow us to gate other features in future without introducing any new code.